radar/tools.js

```diff
@@ -281,70 +281,90 @@ While it lacks a detailed and mature UI and built-in functional testing features
   </p>
 
-
-  <H4>Pros</H4>
+  <p>
+    <strong>Observability testing</strong> (using logs, metrics, and traces) is one example of Shift Right, but Shift Right also includes techniques like:
+  </p>
   <ul>
-    <li><strong>Real-World Validation:</strong> Catches issues that only arise under real user behavior and production loads.</li>
-    <li><strong>Continuous Improvement:</strong> Enables rapid iteration based on user feedback and live metrics.</li>
-    <li><strong>Improved User Experience:</strong> Directly addresses usability, stability, and performance in the environments users actually experience.</li>
+    <li><strong>Canary releases:</strong> Gradually releasing features to a subset of users and monitoring their impact.</li>
+    <li><strong>Chaos testing:</strong> Introducing controlled failures to test system resilience (e.g., with tools like Chaos Monkey).</li>
+    <li><strong>Feature toggles & A/B testing:</strong> Controlling feature rollout and gathering user feedback before full deployment.</li>
+    <li><strong>Service monitoring & alerting:</strong> Using dashboards, logs, and tracing to detect and respond to issues in real time.</li>
   </ul>
 
-  <H4>Cons</H4>
+  <h4>Pros:</h4>
   <ul>
-    <li><strong>Risk of User Impact:</strong> Bugs found in production can negatively affect real users before they are resolved.</li>
-    <li><strong>Requires Robust Monitoring:</strong> Needs advanced monitoring and alerting systems to detect and respond to issues quickly.</li>
-    <li><strong>Dependent on User Base:</strong> Effective shift right testing requires sufficient user activity to generate meaningful feedback and data.</li>
+    <li><strong>Real-world validation:</strong> Ensures the software works as expected in actual usage conditions.</li>
+    <li><strong>Continuous improvement:</strong> Live metrics and user feedback drive ongoing enhancements.</li>
+    <li><strong>Improved user experience:</strong> Allows rapid detection and resolution of issues affecting users.</li>
   </ul>
 
-  <h4>Conclusion:</h4>
+  <h4>Cons:</h4>
   <ul>
-    <li>
-      <strong>Shift Left:</strong> Start testing early (e.g., requirements review, unit testing) to catch issues sooner, reduce late-stage defects, and accelerate development.
-    </li>
-    <li>
-      <strong>Shift Right:</strong> Focus on testing and monitoring in production (e.g., user feedback, A/B testing). While Shift Right covers testing after release, <strong>observability testing</strong> specifically means using logs, metrics, and traces to gain deep insights into system behavior and quickly resolve issues.
-    </li>
+    <li><strong>Risk of user impact:</strong> Bugs in production may affect real users before they are fixed.</li>
+    <li><strong>Robust monitoring required:</strong> Success depends on effective observability and incident response.</li>
```

Comment:
```suggestion
    <li><strong>Requires operational maturity:</strong> Without mature CI/CD pipelines and alerting, the benefits of Shift Right can be limited or delayed.</li>
```

This makes it a bit more "powerfull"